### ISSUE TITLE
Front End: Navbar Bootstrap

### ISSUE THREAD

**Issue by najeeb95** [TechnionYearlyProject/DailyPulseMe#58]
Every website including ours needs a navigation bar and bootstrap helps us achieve that easily through their Navbar.
The Navbar is a wrapper that positions branding, navigation, and other elements in a concise header provided by bootstrap.

Navbars come with built-in support for a handful of sub-components. Choose from the following as needed:

- **.navbar-brand** for your company, product, or project name.
- **.navbar-nav** for a full-height and lightweight navigation (including support for dropdowns).
- **.navbar-toggler** for use with our collapse plugin and other navigation toggling behaviors.
- **.form-inline** for any form controls and actions.
- **.navbar-text** for adding vertically centered strings of text.
- **.collapse.navbar-collapse** for grouping and hiding navbar contents by a parent breakpoint.

Here's an example of a navbar we can use in our website and uses all of the above:

![image](https://user-images.githubusercontent.com/29060164/33233710-04eb9334-d223-11e7-9425-fb7adc424b08.png)

looks good right? well it's code is actually pretty simple:

![image](https://user-images.githubusercontent.com/29060164/33233762-0c6ebdf6-d224-11e7-8eff-50ad5e6bf06b.png)

(the part of the code on the right which we can't see isn't important it's just design settings).

By now all of us have a pretty basic understanding of HTML so the code itself along with built-in sub components which we talked about earlier is pretty self explanitory and there's is one more thing that is not explained here which is the "nav-item" which can't be seen here because it's a screenshot but can be seen [here](https://v4-alpha.getbootstrap.com/components/navbar/?#supported-content) which allows the mouse to hover over the buttons and if we write "nav-item active" it automatically sets them as if it was clicked on.

In summary a navigation bar is critical for our website and we can implement/use templates easily provided by bootstrap, for more info you can click [here](https://v4-alpha.getbootstrap.com/components/navbar/).
